app/src/main/java/com/hedvig/app/feature/onbarding/MoreOptionsRepository.kt

```diff
@@ -0,0 +1,12 @@
+package com.hedvig.app.feature.onbarding
+
+import com.apollographql.apollo.coroutines.await
+import com.hedvig.android.owldroid.graphql.MoreOptionsQuery
+import com.hedvig.app.ApolloClientWrapper
+
+class MoreOptionsRepository(
+    val apolloClientWrapper: ApolloClientWrapper
+) {
+    suspend fun memberId() = apolloClientWrapper.apolloClient.query(MoreOptionsQuery())
+        .await()
+}
```

Comment:
I think there's already a repository that does exactly this - we should re-use that repository 🙂